pyproject.toml

```diff
@@ -69,6 +70,13 @@ flake8-eradicate = "==1.4.0"
 flake8-docstrings = "==1.6.0"
 flake8-isort = "==5.0"
 
+[tool.poetry.group.mnemonic]
+optional = true
+
+[tool.poetry.group.mnemonic.dependencies]
+bip-utils = "*"
```

Comment:
I would vote for leaving `bip-tools` out entirely and maybe replacing it with a few tests against hard-coded mnemonics.